src/NetAppFiles/NetAppFiles/Pool/SetNetAppFilesPool.cs

```diff
@@ -0,0 +1,159 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.NetAppFiles.Common;
+using Microsoft.Azure.Commands.NetAppFiles.Helpers;
+using Microsoft.Azure.Commands.NetAppFiles.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.NetApp;
+using Microsoft.Azure.Management.NetApp.Models;
+
+namespace Microsoft.Azure.Commands.NetAppFiles.Pool
+{
+    [Cmdlet(
+        "Set",
+        ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetAppFilesPool",
+        SupportsShouldProcess = true,
+        DefaultParameterSetName = FieldsParameterSet), OutputType(typeof(PSNetAppFilesPool))]
+    [Alias("Set-AnfPool")]
+    public class SetAzureRmNetAppFilesPool : AzureNetAppFilesCmdletBase
+    {
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = FieldsParameterSet,
```

Comment:
Please also provide the resource id parameter set.

Reply:
I assume for all set cmdLets, will add thanks